Tests/RegexTests/MatchTests.swift

```diff
@@ -915,6 +914,14 @@ extension RegexTests {
       #"\d+(*negative_lookahead: dollars)"#,
       input: "Price: 100 pesos", match: "100")
 
+    // More complex lookaheads
+    firstMatchTest(
+      #"(?=.*e)(?=.*o)(?!.*z)"#,
+      input: "hello", match: "")
```

Comment:
Could you make more input/match pairs? It's a little hard to tell what is matched by `""`